tutorials/ExploringDeutschJozsaAlgorithm/DeutschJozsaAlgorithmTutorial.ipynb

```diff
@@ -514,6 +514,98 @@
     "<video controls width=800 height=600 src=\"state_viz_balanced_annotated.mp4\">Amplitudes change animation for balanced function</video>"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### [Bonus Material] Different function types - what if our function is neither balanced nor constant?\n",
```

Comment:
Looking at the flow of the material, I think this will be best as part V (and make "What's next" part VI)